### PR TITLE
Jj check4update

### PR DESCRIPTION
**Pull request by JorMaister**
Add command checkupdates to check if scipion-app if up to date and also allows the user to update it.

### CODE REVIEW

File: scipion/__main__.py [hunk 1]
```diff
@@ -449,6 +456,8 @@ def main():
     demo | template [PATH] Launches a form based on the *.json.template found either in the PATH
                            or in the pyworkflow/templates directory. If more than one is found,
                            a dialog is raised to choose one. 
+                           
+    checkupdates [FORCE]   Checks for Scipion updates. Use with flag --help to see usage.
```
Comment (pconesa) [inline scipion/__main__.py]: I think FORCE is not clear, I'd use [-h] or [-f to force]

**Comment by azazellochg** [comment]
checkupdates... This is totally opposite of user friendly experience, who has time to type checkupdates? Can we not create super long keywords written by programmers for programmers?

**Comment by pconesa** [comment]
@azazellochg , thanks! what about flipping the behaviour?

"scipion update"   will update

scipion update -check (or just check without the hyphen) will just check it?

I'd also test for non permision error in case installation folder is portected.

**Comment by azazellochg** [comment]
@pconesa that sounds much better, thanks!

**Pull request merged**